src/styles/blocks/card.scss

```
@@ -0,0 +1,102 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $width-card;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform $transition-duration;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: auto;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__rating-value {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 24px;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-title {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    margin: 0;
+    padding: 0;
+    font-size: 16px;
+    font-weight: 700;
+    color: $main-color;
+  }
+
+  &__link {
+    height: 40px;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    display: inline-block;
+    width: 166px;
+    text-align: center;
+    color: $white_color;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $blue-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    box-sizing: border-box;
+
+    &:hover {
+      background: $white_color;
+      color: $blue-color;
+      border-color: $blue-color;
+      box-sizing: border-box;
```

```suggestion

    &:hover {
      background: $white_color;
      color: $blue-color;
      border-color: $blue-color;
```
You've already set the box-sizing for all element, there is no need to change it anymore.
Also, it would be great to add transition for button, so its hover effect works smoothly too